modin/logging/config.py

```diff
@@ -109,35 +126,39 @@ def configure_logging():
     )
     logfile.setFormatter(formatter)
     logger.addHandler(logfile)
+    logger.setLevel(log_level)
 
-    if LogMode.get() == "enable_api_only":
-        logger.setLevel(logging.INFO)
-    logger.setLevel(logging.DEBUG)
+    return logger
+
+
+def configure_logging():
+    """Configure Modin logging by setting up directory structure and formatting."""
+    global __LOGGER_CONFIGURED__
+    logger = logging.getLogger("modin.logger.default")
```

Comment:
it looks like `logger` is set twice, once in 137 and once in 140. is this necessary?

Reply:
good catch, it won't hurt but it isn't needed